core/src/module/usModule.cpp

```diff
@@ -119,15 +121,37 @@ bool Module::IsLoaded() const
 
 void Module::Start()
 {
-
+  MutexLockingStrategy::Lock(this->d);
```

Comment:
This multi-threading issue will have to be fixed for 3.0 sooner or later. I'd prefer sooner, so I will fix this one now within this PR.